src/gui/scene/Renderer.js

```diff
@@ -81,14 +181,33 @@ class Renderer {
                 this.#scene.remove(this.#camera.getInternalObject());
             }
             this.#scene.add(obj.getInternalObject());
-            this.#camera = obj;
         } else if (obj instanceof AnchoredGroup) {
-            this.#scene.add(obj.getInternalObject());
+            this.#globalScale.getNode().add(obj.getInternalObject());
         } else if (obj instanceof THREE.Object3D) {
-            this.#scene.add(obj)
+            this.#globalScale.getNode().add(obj)
         }
     }
 
+    /**
+     * 
+     * @returns {boolean}
+     */
+    isInWebXRMode() {
+        return this.webXRAvailable() && this.#renderer.xr.isPresenting;
+    }
+
+    webXRAvailable() {
+        return this.#renderer.xr && this.#renderer.xr.enabled === true;
+    }
+
+    /**
+     * 
+     * @param {Camera} camera 
+     */
+    setCamera(camera) {
```

Comment:
we explicitly set a camera here, so we can choose the camera outside of the renderer. In the future there might be other cameras depending on the hardware platform we're working on. By seperating camera and renderer we can add those more simply. This does get a bit weird in this case, since the webxr camera is managed by the three.js renderer's xr manager. So even if we don't switch the cameras three.js just overrides our choice of camera internally.
The proposed setup will enable us to create an arview camera for the native ios app, a camera with controls for the remote desktop variant and a camera for webxr. At this moment however the first two cameras use the same default class with different configurations.